app/src/main/java/cm/aptoide/pt/ads/AdsUserPropertyManager.java

```diff
@@ -36,4 +37,14 @@ public void start() {
         .subscribe(created -> {
         }, error -> crashReport.log(error));
   }
+
+  public Completable setUp() {
+    return installedRepository.isInstalled(WALLET_PACKAGE)
+        .first()
+        .observeOn(ioScheduler)
+        .distinctUntilChanged()
+        .flatMapSingle(__ -> moPubAdsManager.getAdsVisibilityStatus())
+        .doOnNext(moPubAnalytics::setAdsVisibilityUserProperty)
+        .toCompletable();
+  }
```

Comment:
why are we keeping two very similar methods with repeated code? can't we merge both?